ingestion-beam/checkstyle

```diff
@@ -0,0 +1 @@
+../checkstyle/
```

Comment:
softlinks are necessary for checkstyle to work in both submodule directories and the parent module directory.